website/meta/universe.json

```diff
@@ -2000,6 +2000,44 @@
             },
             "category": ["pipeline"],
             "tags": ["syllables", "multilingual"]
+        },
+        {
+            "id": "gobbli",
+            "title": "gobbli",
+            "slogan": "Deep learning for text classification doesn't have to be scary",
+            "description": "gobbli is a Python library which wraps several modern deep learning models in a uniform interface that makes it easy to evaluate feasibility and conduct analyses. It leverages the abstractive powers of Docker to hide nearly all dependency management and functional differences between models from the user. It also contains an interactive app for exploring text data and evaluating classification models.",
```

Comment:
If possible, could you add a sentence or slightly rephrase it so it mentions that it integrates with spaCy or that it provides easy access to spaCy's text classifier? Just to make sure it's immediately obvious why this is in the universe and people don't get confused.

Reply:
Good suggestion, added in latest commit. Happy to incorporate any other feedback as well. Thanks!

Reply:
Great, thanks! 👍